src/web/components/Pillars.tsx

```diff
@@ -263,6 +263,8 @@ export const Pillars: React.FC<{
 							isSelected && forceUnderline,
 							showDivider && pillarDivider,
 						]}
+						id={i === 0 ? 'navigation' : undefined}
+						aria-label={i === 0 ? 'Navigation' : undefined}
```

Comment:
We tell the reader they are at the start of the navigation section using this label